CoreJs-part7-css/CoreJs-part7.css

```diff
@@ -0,0 +1,76 @@
+/***************************************************************************************************************************
+1. Write a CSS selector to target all <p> elements with a class of "highlight" and give them a background color of yellow.*/
+.highlight {
```

Comment:
You want to target **p** element with _highlight_ class. So you should have `p.highlight` selector.

Because you could have a div which you don't want to target, example:
`<div class="highlight"></div>`